samples/app-checkin-location/python/app.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
Please update the copyright header on the missing files as shown below:

#Copyright (c) Microsoft Corporation. All rights reserved.
#Licensed under the MIT License.